tests/controller/elementControllerTest.js

```diff
@@ -125,15 +131,21 @@ before(async function () {
 
     // Create 2 QRcodes
     idQR1 = await QR.create({
-        code: "QRcode1"
+        code: codeQR1
     }).then((doc) => {
         return doc._id;
     });
     idQR2 = await QR.create({
-        code: "QRcode2"
+        code: codeQR2
     }).then((doc) => {
         return doc._id;
     });
+    await QR.create({
```

Comment:
idQR3 never assigned

Reply:
No needed. Just need to add a QR in the database